### DISCUSSION TITLE
Timestamping opportunities (last-updated)

### DISCUSSION THREAD

**Discussion by FloriSpoelstra** [alkem-io/alkemio#371] [Ideas] [answered]
I would love to have a "last updated" date on the opportunities page. So people can see how up-to-date the information is. And it would also be great to show it on the opportunities overview page. Like they are doing here: https://www.winnovatie.nl/innovaties

**Comment by techsmyth** [comment]
Nice suggestion!

Key question then is what actions would trigger updating this timestamp? Editing the Context information? Adding a new reference? Updating Tags?

**Comment by FloriSpoelstra** [comment]
I think any of those should trigger the timestamp. It would show that the info is maintained.

**Comment by techsmyth** [comment, marked as answer]
Captured on the backlog under https://github.com/alkem-io/client-web/issues/958 - which is part of the whole challenge / opportunity pages refresh epic.

**Reply by FloriSpoelstra** [reply]
Thanks!
